lxd/network/network_utils.go

```diff
@@ -1038,8 +1039,14 @@ func VLANInterfaceCreate(parent string, vlanDevice string, vlanID string) (bool,
 		return false, errors.Wrapf(err, "Failed to bring up parent %q", parent)
 	}
 
+	// Prepare variadic argument string
+	var args []string
```

Comment:
@TimRots little nit here, but this could just be:

```
args :=  []string{"link", "add", "link", parent, "name", vlanDevice, "up", "type", "vlan", "id", vlanID}
```

Reply:
Good point, updated the PR